test/Sema/diag_defer_block_end.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %target-swift-frontend -typecheck -verify %s
```

Comment:
You can take care of this in a future commit, but this could actually be `// RUN: %target-typecheck-verify-swift` without explicit options or `%s`.